extension/azureauthextension/README.md

```diff
@@ -11,10 +11,11 @@
 <!-- end autogenerated section -->
 
 This extension implements both `extensionauth.HTTPClient` and `extensionauth.Server`, so it can be used in both exporters and receivers.
-**Currently, this part is still not implemented.**
 
 Additionally, the extension also implements `azcore.TokenCredential` so that Azure components can get the token by running the function `GetToken`. If the component supports HTTP client, then this should not be necessary, as the token will be placed in the authorization header.
 
+> **WARNING**: Currently, the token is not cached, so if you use the extension as an implementation of `extensionauth.HTTPClient` and `extensionauth.Server`, instead of `azcore.TokenCredential`, use it with caution. The requests that go through the extension for this purpose need to have the `Host` header. Read more about [request headers](https://learn.microsoft.com/en-us/azure/azure-app-configuration/rest-api-headers#request-headers).
```

Comment:
what type of caution?

Reply:
I have added this to this part of the README file:

> Each of these requests to get a token will cause a new request being sent to Azure Identity. Under high load, be careful to not hit the service [request limits](https://learn.microsoft.com/en-us/entra/identity/users/directory-service-limits-restrictions).

Is it more clear now?

I expect the main use of this extension will be as `azcore.TokenCredential` and not so much as a server or client authenticator. So I am not considering this issue as a top priority to get fixed at the moment, but something that needs to be done in the future.

Reply:
this is much better, thank you.